src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -190,9 +190,14 @@ internal static void UpdateBackground(this EditText platformView, IView view)
 				mauiDrawable.Dispose();
 			}
 
-			var previousDrawable = platformView.Background;
+			if(platformView.Background is LayerDrawable layerDrawable)
+			{
+				platformView.Background = null;
+				layerDrawable.Dispose();
+			}
+
 			var backgroundDrawable = paint!.ToDrawable(platformView.Context);
-			LayerDrawable layer = new LayerDrawable(new Drawable[] { backgroundDrawable!, previousDrawable! });
+			LayerDrawable layer = new LayerDrawable(new Drawable[] { backgroundDrawable!, defaultBackground! });
```

Comment:
Try to avoid the use of !, are we sure this is not null, better wrap on an if that an NRE. What happens is null? do we need the new layer, do we exit the method? Do we create a new later with no drawable contents?